src/php/binaries.rs

```diff
@@ -31,7 +31,7 @@ pub(crate) fn all() -> HashMap<PhpVersion, PhpBinary> {
 
     merge_binaries(binaries_from_dir(PathBuf::from("/usr/bin")), &mut binaries);
     merge_binaries(binaries_from_dir(PathBuf::from("/usr/sbin")), &mut binaries);
-    merge_binaries(binaries_from_dir(PathBuf::from("/usr/local/Cellar/php/*/")), &mut binaries);
+    merge_binaries(binaries_from_dir(PathBuf::from("/usr/local/Cellar/php*/*/")), &mut binaries);
```

Comment:
Not the best idea probably, because of other tools whose name starts with `php`:
```
$ls /usr/local/Cellar/php*

/usr/local/Cellar/php:
7.4.9

/usr/local/Cellar/php-code-sniffer:
3.4.2

/usr/local/Cellar/php-cs-fixer:
2.16.4

/usr/local/Cellar/php@7.3:
7.3.21
```
Please check my solution in #17 (which now has a merge conflict 😁

Reply:
I'll check it right away :D